<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QAnimationStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#stackWidget{
background-color:rgba(255, 255, 255,0);
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhEmailCharactersOnly</set>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainwindow">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>531</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
border-radius:30px;
background:white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>281</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label{
border-image:url(:/image/bg2.jpg);
border-top-left-radius:30px;
border-bottom-left-radius:30px;
padding:2px 4px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>0</y>
         <width>41</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#closeButton{
border-top-right-radius:30px;
}
#closeButton:hover{
background:red;
}</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_login_switch">
       <property name="geometry">
        <rect>
         <x>344</x>
         <y>40</y>
         <width>38</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>25</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_login_switch{
border:none;
border-image:url(:/image/login.png);
border-radius:5px;
}
#btn_login_switch:hover{
background:rgb(121, 121, 121);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="helloLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Brush Script MT</family>
         <pointsize>30</pointsize>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#helloLabel{
color:white;
}</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Good Evening&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>452</x>
         <y>0</y>
         <width>38</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>29</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_2{
border:none;
}
#pushButton_2:hover{
background:rgb(0, 170, 255);
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QAnimationStackedWidget" name="stackedWidget_2">
       <property name="geometry">
        <rect>
         <x>284</x>
         <y>79</y>
         <width>231</width>
         <height>191</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="loginPage">
        <widget class="QLineEdit" name="lineEdit_username">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>211</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#lineEdit_username{
border:3px solid black;
font-size:15px;
border-radius:10px;
}
#lineEdit_username:hover{
border:4px solid black;
padding:5px;
}</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Input Account</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>78</y>
           <width>211</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#lineEdit_password{
border:3px solid black;
font-size:15px;
border-radius:10px;
}

#lineEdit_password:hover{
border:4px solid black;
padding:5px;
}</string>
         </property>
         <property name="maxLength">
          <number>16</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Input Password</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_login">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>139</y>
           <width>191</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_login{
background:black;
color:white;
font:normal 400 20px;
border-radius:10px;
border:none;
}
#btn_login:hover{
background:white;
color:black;
font:normal 400 20px;
border-radius:10px;
border:4px solid black;
}</string>
         </property>
         <property name="text">
          <string>Sign in</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S, Ctrl+S</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_signal">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#label_signal{
color:red;
}</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="registerPage">
        <widget class="QLineEdit" name="lineEdit_newpass">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>51</y>
           <width>211</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#lineEdit_newpass{
border:3px solid black;
font-size:15px;
border-radius:10px;
}
#lineEdit_newpass:hover{
border:4px solid black;
padding:5px;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Your Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_newname">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>9</y>
           <width>211</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#lineEdit_newname{
border:3px solid black;
font-size:15px;
border-radius:10px;
}

#lineEdit_newname:hover{
border:4px solid black;
padding:5px;
}</string>
         </property>
         <property name="placeholderText">
          <string>New Account</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_surepass">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>93</y>
           <width>211</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#lineEdit_surepass{
border:3px solid black;
font-size:15px;
border-radius:10px;
}
#lineEdit_surepass:hover{
border:4px solid black;
padding:5px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Ensure Password</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_sureadd">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>148</y>
           <width>191</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_sureadd{
background:black;
color:white;
font:normal 400 20px;
border-radius:10px;
border:none;
}
#btn_sureadd:hover{
background:white;
color:black;
font:normal 400 20px;
border-radius:10px;
border:4px solid black;
}</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_signal_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>130</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#label_signal_2{
color:red;
}</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="btn_new_switch">
       <property name="geometry">
        <rect>
         <x>416</x>
         <y>40</y>
         <width>36</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_new_switch{
border:none;
border-image:url(:/image/register.png);
border-radius:5px;
}
#btn_new_switch:hover{
background:rgb(121, 121, 121);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>389</x>
         <y>40</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>281</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_2{
border-image:url(:/image/logo.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>btn_login_switch</zorder>
      <zorder>helloLabel</zorder>
      <zorder>closeButton</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>stackedWidget_2</zorder>
      <zorder>btn_new_switch</zorder>
      <zorder>line</zorder>
      <zorder>label_2</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="block"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">qanimationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
